examples-new/native-authentication-exchange/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SECRET_KEY": "replace me with a random string",
+  "NYLAS_OAUTH_API_ID": "replace me with the API ID from Nylas",
```

Comment:
Could you use a different name in all of them please? On our end, we talk to customers in terms of a Nylas `client_id` and `client_secret`, so I'd like to keep that naming convention consistent. Sorry for not catching it earlier!